packages/manager/cypress/e2e/core/cloudpulse/linode-widget-verification.spec.ts

```diff
@@ -273,34 +290,32 @@ describe('Integration Tests for Linode Dashboard ', () => {
             );
           });
 
-          //validate the widget linegrah is present
```

Comment:
May be //Verify that the widget line graph is displayed ?

Reply:
fixed